tailwind.config.js

```diff
@@ -1,11 +1,55 @@
 /** @type {import('tailwindcss').Config} */
+const defaultTheme = require('tailwindcss/defaultTheme');
+
 module.exports = {
   content: [
-    "./pages/**/*.{js,ts,jsx,tsx}",
-    "./components/**/*.{js,ts,jsx,tsx}",
+    './pages/**/*.{js,ts,jsx,tsx}',
+    './components/**/*.{js,ts,jsx,tsx}',
   ],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {
+        sans: ['var(--font-poppins)', ...defaultTheme.fontFamily.sans],
+      },
+      colors: {
+        white85: 'rgba(255, 255, 255, 0.85);',
+      },
+      fontSize: {
+        h2: [
+          '32px',
+          {
+            lineHeight: '48px',
+            letterSpacing: '0.02em',
+            fontWeight: '400',
+          },
+        ],
+        p: [
+          '16px',
+          {
+            lineHeight: '24px',
+            letterSpacing: '0.01em',
+            fontWeight: '400',
+          },
+        ],
+      },
+      backgroundImage: {
+        template: 'linear-gradient(33.75deg, #00B7BB 0%, #148EFF 100%)',
```

Comment:
It should not be called template as this gradient color can be used somewhere else.